neurom/fst/_core.py

```diff
@@ -85,7 +85,7 @@ def make_neurites(rdw):
     # One pass over sections to build nodes
     nodes = tuple(Section(section_id=i,
                           points=rdw.data_block[sec.ids],
-                          section_type=_TREE_TYPES[sec.ntype])
```

Comment:
The reason dict is making a difference is here: _TREE_TYPES used to be accessed with sec.ntype used as the index in the _TREE_TYPES list. It was almost working by luck and was assuming consecutivity of the defined numbers. Doing _TREE_TYPES[custom = 7] should have meant that custom should be at the 8th position of the _TREE_TYPES list.

Reply:
Ok, but I don't see why that requires renumber the constants?

Reply:
Because now sec.ntype is now used as the key of the dict. And sec.ntype is the value from the file you're reading so it has to be in sync.